identity/identity.go

```diff
@@ -20,6 +20,22 @@ import (
 	"github.com/ory/kratos/x"
 )
 
+// swagger:model identityState
+type State string
```

Comment:
Sorry to bother you with this, but there is even an enum definition
https://github.com/ory/keto/blob/0ac4fe1667ee1edd133fe3f117e204e3144f6c78/internal/expand/tree.go#L15-L23
You will have to move the const definitions right below the type definition IIRC, then it should work. FYI @aeneasr

Reply:
We can use enum in the property field - commented in the PR below

Reply:
Why would we not define it once for the `State` type and it just works everywhere?

Reply:
I don't think go-swagger supports that

Reply:
It does, the above referenced definition is translated into https://github.com/ory/keto/blob/0ac4fe1667ee1edd133fe3f117e204e3144f6c78/.schema/swagger.json#L1025-L1032
Added by this PR https://github.com/go-swagger/go-swagger/pull/2176

Reply:
I tried it and it does work. :)